docs/elements.md

```diff
@@ -102,6 +102,9 @@ compatibility, but they should not be used in new questions.
 
 A `pl-multiple-choice` element selects **one** correct answer and zero or more
 incorrect answers and displays them in a random order as radio buttons.
+Duplicate answer choices (string equivalents) are not permitted in the
+`pl-multiple-choice` element, and an exception will be raised upon question
+generation if two (or more) choices are identical.
```

Comment:
This may be a problem for dynamically generated options that happen to be identical. Can they be silently ignored?

Reply:
This is actually the intended behavior change (see https://github.com/PrairieLearn/PrairieLearn/issues/1228). We _could_ add a flag that enables silently ignoring identical options (defaulted to false), but the purpose of this is to loudly fail. I don't know why an instructor would want to allow identical options. Perhaps if there was a way to raise a warning that got displayed to the instructor only and not students?

Reply:
I'm thinking something like:

```
<pl-answer>{{params.a}}</pl-answer>
<pl-answer>{{params.b}}</pl-answer>
...
```

Where `a` and `b` may randomly end up having the same value. I'd imagine they'd be deduplicated (i.e., only one would appear). I indeed see the discussion in the original issue, and will mark this conversation as resolved, if this comes up again we can have a new issue. It probably makes sense to an instructor that wants to create these random options to have some type of handling of duplicate options.